cmake/common.cmake

```diff
@@ -584,8 +608,6 @@ function(add_picobench name)
 
   add_executable(${name} ${PARSED_ARGS_SRCS})
 
-  add_lvi_mitigations(${name})
```

Comment:
I'm assuming that removing this causes the performance improvement for all our micro benchmarks?